lib/src/formatter.dart

```diff
@@ -173,9 +173,7 @@ class SimpleFormatter implements ReportFormatter {
     var tableWidth = max(_summaryLength, longest + largestCountGuess);
     var pad = tableWidth - longest;
     var line = ''.padLeft(tableWidth, '-');
-    out.writeln(line);
-    out.writeln('Counts');
-    out.writeln(line);
+    out..writeln(line)..writeln('Counts')..writeln(line);
```

Comment:
It looks like an issue with `dartfmt` rather than with the cascade lint.

If `dartfmt` put cascades on new lines it will be simpler to read.